### PR TITLE
build(deps): switch from yarn to npm

### PR DESCRIPTION
**Pull request by caugner**
Migrates bob to use npm instead of yarn, following the vote in https://github.com/orgs/mdn/discussions/108.

Steps:
1. Removed `yarn.lock`.
2. Ran `npm install`.
3. Replaced or removed all occurrences of `yarn` across the repository.

### CODE REVIEW

**Comment by schalkneethling** [comment]
@caugner This looks good to me! Thanks. There is currently a conflict that needs resolving. (also, https://github.com/mdn/bob/pull/800 😄 )

**Comment by caugner** [comment]
@schalkneethling ✅ Conflict resolved.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -12,10 +12,10 @@ jobs:
                   node-version: '12'
```
Comment (schalkneethling) [inline .github/workflows/test.yml:12]: Should we bump this to 16?

**Review by schalkneethling — COMMENTED** [review]
One question regarding the node version for the workflow.

File: .github/workflows/test.yml (hunk 1 above)
Comment (caugner) [reply .github/workflows/test.yml:12]: I'd do this in a separate PR, but yes.

**Review by schalkneethling — APPROVED** [review]

**Pull request merged**